types/react-responsive-tabs.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module 'react-responsive-tabs' {
```

Comment:
I don't know the rationale behind this but at least in my case this results in 
eslint error:
`Parsing error: Cannot read property 'map' of undefined`

```suggestion
declare module 'react-responsive-tabs';

```